src/ray/util/logging.h

```diff
@@ -34,12 +34,15 @@ namespace ray {
                         << __FILE__ << __LINE__          \
                         << " Check failed: " #condition " "
 
+#define RAY_DCHECK(condition) RAY_CHECK(condition)
```

Comment:
Make it no-op if NDEBUG?  This is basically how DCHECK is defined in glog.

Reply:
Fixed.